Missing some import entries in OSGI headers

## Environment
**Liquibase Version**: 3.10.1

**Liquibase Integration & Version**: osgi apache servicemix 7.0.1 (karaf 4.0.9)

**Liquibase Extension(s) & Version**: 

**Database Vendor & Version**: PostgreSQL 12

**Operating System Type & Version**: Fedora Linux 32

## Description
There are still some problems with OSGI headers even if issue #1108 was fixed. I have tried to execute liquibase changeset under Apache ServiceMix but it failed with an exception :
java.lang.NoClassDefFoundError: javax/activation/DataSource
	at java.lang.ClassLoader.defineClass1(Native Method) ~[?:1.8.0_252]
	at java.lang.ClassLoader.defineClass(ClassLoader.java:756) ~[?:1.8.0_252]
	at org.apache.felix.framework.BundleWiringImpl$BundleClassLoader.defineClass(BundleWiringImpl.java:2310) ~[?:?]
	at org.apache.felix.framework.BundleWiringImpl$BundleClassLoader.findClass(BundleWiringImpl.java:2094) ~[?:?]
	at org.apache.felix.framework.BundleWiringImpl.findClassOrResourceByDelegation(BundleWiringImpl.java:1518) ~[?:?]
	at org.apache.felix.framework.BundleWiringImpl.access$200(BundleWiringImpl.java:79) ~[?:?]
	at org.apache.felix.framework.BundleWiringImpl$BundleClassLoader.loadClass(BundleWiringImpl.java:1958) ~[?:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351) ~[?:1.8.0_252]
	at liquibase.osgi.OSGiPackageScanClassResolver.find(OSGiPackageScanClassResolver.java:39) ~[?:?]
	at liquibase.servicelocator.DefaultPackageScanClassResolver.findImplementations(DefaultPackageScanClassResolver.java:89) ~[?:?]
	at liquibase.servicelocator.ServiceLocator.findClassesImpl(ServiceLocator.java:228) ~[?:?]
	at liquibase.servicelocator.ServiceLocator.findClasses(ServiceLocator.java:203) ~[?:?]
	at liquibase.database.DatabaseFactory.<init>(DatabaseFactory.java:28) ~[?:?]
	at liquibase.database.DatabaseFactory.getInstance(DatabaseFactory.java:46) ~[?:?]
	at tech.pantheon.chronos.liquibase.LiquibaseUtils.createDatabase(LiquibaseUtils.java:73) ~[?:?]
	at tech.pantheon.chronos.liquibase.LiquibaseUtils.createLiquibase(LiquibaseUtils.java:56) ~[?:?]
	at tech.pantheon.chronos.liquibase.LiquibaseDrop.performDrop(LiquibaseDrop.java:36) ~[?:?]
	at tech.pantheon.chronos.dev.command.LiquibaseDropAllCommand.execute(LiquibaseDropAllCommand.java:38) ~[?:?]
	at org.apache.karaf.shell.impl.action.command.ActionCommand.execute(ActionCommand.java:83) ~[!/:?]
	at org.apache.karaf.shell.impl.console.osgi.secured.SecuredCommand.execute(SecuredCommand.java:67) ~[!/:?]
	at org.apache.karaf.shell.impl.console.osgi.secured.SecuredCommand.execute(SecuredCommand.java:87) ~[!/:?]
	at org.apache.felix.gogo.runtime.Closure.executeCmd(Closure.java:480) ~[!/:?]
	at org.apache.felix.gogo.runtime.Closure.executeStatement(Closure.java:406) ~[!/:?]
	at org.apache.felix.gogo.runtime.Pipe.run(Pipe.java:108) ~[!/:?]
	at org.apache.felix.gogo.runtime.Closure.execute(Closure.java:182) ~[!/:?]
	at org.apache.felix.gogo.runtime.Closure.execute(Closure.java:119) ~[!/:?]
	at org.apache.felix.gogo.runtime.Closure.executeCmd(Closure.java:480) ~[!/:?]
	at org.apache.felix.gogo.runtime.Closure.executeStatement(Closure.java:406) ~[!/:?]
	at org.apache.felix.gogo.runtime.Pipe.run(Pipe.java:108) ~[!/:?]
	at org.apache.felix.gogo.runtime.Closure.execute(Closure.java:182) ~[!/:?]
	at org.apache.felix.gogo.runtime.Closure.execute(Closure.java:119) ~[!/:?]
	at org.apache.felix.gogo.runtime.CommandSessionImpl.execute(CommandSessionImpl.java:94) ~[!/:?]
	at org.apache.karaf.shell.impl.console.ConsoleSessionImpl.run(ConsoleSessionImpl.java:274) [!/:?]
	at org.apache.karaf.shell.impl.console.osgi.DelayedStarted.run(DelayedStarted.java:75) [!/:?]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_252]
Caused by: java.lang.ClassNotFoundException: javax.activation.DataSource not found by org.liquibase.core [244]
	at org.apache.felix.framework.BundleWiringImpl.findClassOrResourceByDelegation(BundleWiringImpl.java:1550) ~[?:?]
	at org.apache.felix.framework.BundleWiringImpl.access$200(BundleWiringImpl.java:79) ~[?:?]
	at org.apache.felix.framework.BundleWiringImpl$BundleClassLoader.loadClass(BundleWiringImpl.java:1958) ~[?:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351) ~[?:1.8.0_252]
	... 35 more

At first I have tried to use command "bundle:dynamic-import" for liquibase-core bundle and everything was working with this. As the second step I have tried to edit manually MANIFEST.MF for liquibase-core.jar. I have added these entries to "Import-Package" section :
javax.activation;version="[1.1,1.1]",javax.xml.bind;version="[2.2.1,2.2.1]",javax.xml.bind.annotation.adapters;version="[2.2.1,2.2.1]",javax.xml.bind.annotation;version="[2.2.1,2.2.1]". (I set one specific version for this quick test). With this update I was also able to execute liquibase changesets.

## Steps To Reproduce
We have an internal library that calls liquibase, our createDatabase  method (see exception stack trace)  contains this code:
  DatabaseConnection liquibaseConnection = new JdbcConnection(c);
    Database database
            = DatabaseFactory.getInstance().findCorrectDatabaseImplementation(liquibaseConnection);

## Additional Context
We don't have any problem with liquibase 3.8.0



┆Issue is synchronized with this [Jira Bug](https://datical.atlassian.net/browse/LB-365) by [Unito](https://www.unito.io)
┆fixVersions: Community 4.x

--- a/liquibase-core/pom.xml
+++ b/liquibase-core/pom.xml
@@ -171,6 +171,7 @@
                     <instructions>
                         <Bundle-SymbolicName>org.liquibase.core</Bundle-SymbolicName>
                         <Import-Package>
+                            javax.activation*;resolution:=optional,
                             javax.servlet.*;version="[2.6,4)";resolution:=optional,
                             javax.sql.*,
                             org.osgi.*,

